packages/core/src/hydration/cleanup.ts

```diff
@@ -53,6 +53,16 @@ export function removeDehydratedViews(lContainer: LContainer) {
   lContainer[DEHYDRATED_VIEWS] = retainedViews;
 }
 
+export function removeDehydratedViewList(deferBlock: DehydratedDeferBlock) {
+  if (deferBlock.lContainer[DEHYDRATED_VIEWS] === null) return;
+  const parentLView = deferBlock.lContainer[PARENT];
+  const renderer = parentLView[RENDERER];
+  for (const view of deferBlock.lContainer[DEHYDRATED_VIEWS]) {
```

Comment:
nit (for a minor code-size optimization):
```suggestion
  const {lContainer} = deferBlock;
  const dehydratedViews = lContainer[DEHYDRATED_VIEWS];
  if (dehydratedViews === null) return;
  const parentLView = lContainer[PARENT];
  const renderer = parentLView[RENDERER];
  for (const view of dehydratedViews) {
```

Reply:
The irony here is that I had it this way and then decided I didn't need to assign a variable for `dehydratedViews` to shorten up the code.